unpackaged/config/install_emulator/classes/EDADemoReleaseGate.cls

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
Please move this and the other classes to a new folder unique to them like `unpackaged/config/release_gating`. You can then edit the cumulusci.yml to add a task to deploy this and incorporate it into the qa config area.

Reply:
Moved to release_gating folder, added new cumulusCI task